### ISSUE TITLE
Crash using 11.05r2 and 0.44

### ISSUE THREAD

**Issue by Semirben** [skiselkov/BetterPushbackC#132]
Hi Saso,

I tried v0.44 today with X-Plane 11.05r2 and noticed that X-Plane crashes when trying to load BetterPushback. Here is the changelog for 11.05r2 and my X-Plane Log is attached
```
- XSG-3447 Lists KMCO STARS-SIDS as NONE.
- XPD-8136, XPD-8193 Removed Aerosoft airports for which a better Gateway version exists.
- XPD-8176 Removed undesirable files in 11.05 custom airports
```
Interpreting the log output it seems like the plugin has problems with interpreting runway headings.

Regards,
Daniel
[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/1264964/Log.txt)

**Comment by skiselkov** [comment]
You cannot believe how much I love you for using Linux, because there I can actually properly translate the stack trace.

**Comment by skiselkov** [comment]
Which airport were you located at when this crash occurred? And were you using default scenery, or custom? If custom, which one?

**Comment by Semirben** [comment]
The crash occured when launching X-Plane. So I don't even get to the main menu. But the last airport I was before this problem occured was London Gatwick (EGKK), default scenery, with the default Cirrus Vision SF50

**Comment by skiselkov** [comment]
Can you please try this binary? It's not a full plugin, just the Linux binary itself. Place it into the `64` directory in the plugin (replacing the `lin.xpl` that's there already).
[lin.xpl.zip](https://github.com/skiselkov/BetterPushbackC/files/1267107/lin.xpl.zip)

**Comment by Semirben** [comment]
With this binary everything is working fine. Thank you very much :)

**Comment by skiselkov** [comment]
Ok, fix coming in next build.

**Comment by skiselkov** [comment]
Fixed in libacfutils commit 2d82fb29.
